packages/elastic-apm-synthtrace/src/scripts/run.ts

```diff
@@ -22,9 +24,23 @@ function options(y: Argv) {
       string: true,
     })
     .option('target', {
-      describe: 'Elasticsearch target, including username/password',
+      describe: 'Elasticsearch target',
+      string: true,
+    })
+    .option('cloudId', {
+      describe:
+        'Provide connection information and will force APM on the cloud to migrate to run as a Fleet integration',
+      string: true,
+    })
+    .option('username', {
+      describe: 'Basic authentication username',
+      string: true,
       demandOption: true,
+    })
+    .option('password', {
+      describe: 'Basic authentication password',
       string: true,
+      demandOption: true,
     })
```

Comment:
I think something like `--target=http://admin:changeme@localhost` is pretty easy to use, I'd rather not require separate authentication options. Can we make them optional and throw an error if cloudId is used instead of target? I think `yargs` also allows for either/or options w/ `argv.conflicts()` (for target/cloudId).

Reply:
Both should still work :)